pkg/packagekit/updates.scss

```diff
@@ -70,36 +70,11 @@ table.listing-ct thead th {
 
 .ct-table tr {
     td.type {
-        min-width: 10ch;
-    }
-
-    @media screen and (min-width: 641px) {
-        /* Desktop: Truncate version by default and untruncate on hover */
-        td.version {
-            width: 18ex;
-            /* Tables are stretchy, so we *also* need the max-width */
-            max-width: 18ex;
-            white-space: nowrap;
-            overflow: hidden;
-            text-overflow: ellipsis;
-
-            /* Expand version string when hovering */
-            &:hover {
-                overflow: visible;
-
-                .truncating {
-                    background: linear-gradient(to left, rgba(237, 248, 255, 0), var(--ct-color-list-hover-bg) 3em);
-                    position: relative;
-                    padding: 0.5em 4em 0.5em 0;
-                }
-            }
+        .severity-icon {
+            margin-right: var(--pf-global--spacer--xs);
```

Comment:
can this selector be made simpler?
like:
```
.ct-table tr {
    .severity-icon {
...
```
? I guess that the `td.type` is not necessary there, is it? And if is, it can be still squashed into one-line selector.
```

Reply:
done